packages/material-ui/src/utils/helpers.js

```diff
@@ -12,10 +12,6 @@ export function capitalize(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
-export function contains(obj, pred) {
```

Comment:
Dead code, something we could have spotted with 💯% test coverage. But that's part of the tradeoff.

Reply:
It doesn't change the bundle size. Tree shaking works 😍 .